sdk/core/amqp-common/changelog.md

```diff
@@ -1,3 +1,7 @@
+### 2019-5-6 1.0.0-preview.4
+
+- Moved repo from https://github.com/Azure/amqp-common-js to https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/core/amqp-common [Issue 1208](https://github.com/Azure/azure-sdk-for-js/issues/1208)
```

Comment:
Ramya mentioned in the email that there is a bug fix for this issue https://github.com/Azure/amqp-common-js/issues/59 
I think this should also be added in the changelog, when the bugfix is there, sometime before releasing the package

Reply:
The issue is migrated to https://github.com/Azure/azure-sdk-for-js/issues/2706 and has not been closed/verified explicitly. We might want to wait before adding that.

Reply:
ok

Reply:
It will be fixed before 1.0.0 likely, so preview.4 would be good.

Reply:
@ramya-rao-a Can you review the changelog for preview .4 ?
 - The planned date of the release
- The fixes we're planning to release

Reply:
The fix for #2706 has been committed, see #2703. Since the bits we are shipping will have the fix, it should be called out in the changeling

Reply:
Also, the move doesn't affect our end user, so it doesn't need to be in the changelog.

Reply:
Got it, could you share precise summary of details on the fix to add into the changelog?
Is statement 'Removed @types/async-lock from dev dependencies and added to regular dependencies' sufficient? (with link to PR?)

Reply:
> Removed @types/async-lock from dev dependencies and added to regular dependencies

That only says what was done, but not why.

> Updated `dependencies` to include `@types/async-lock` to fix Typescript compile error "TS7016: Could not find a declaration file for module 'async-lock'".